pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>org.testobject</groupId>
   <artifactId>espressorunner</artifactId>
-  <version>0.2.1-SNAPSHOT</version>
```

Comment:
I think we should go to 0.3.0 as this is a larger change

Reply:
done